lib/ui/plugins/callback_cache.cc

```diff
@@ -126,14 +126,14 @@ void DartCallbackCache::LoadCacheFromDisk() {
   }
   std::string cache_contents{std::istreambuf_iterator<char>(input),
                              std::istreambuf_iterator<char>()};
-  Document d;
-  d.Parse(cache_contents.c_str());
-  if (d.HasParseError() || !d.IsArray()) {
+  Document document;
```

Comment:
let's not have these rename in this PR. seems not related.

Reply:
Jup, i‘ll remove the refactoring when i have time. And yes, this will reintroduce a Memoryleak, which has been fixed. If you want it to be a real fix instead of a Workaround feel free to close this PR and i will take the time to create a real fix. But in my opinion the crash is more severe than a memoryleak.

Reply:
I think the same